Issue 2045. Deserialization JSON from Db

Nastsi opened the pull request.

Nastsi committed:
Merge remote-tracking branch 'refs/remotes/origin/master' into issue-2045

Nastsi committed:
issue#2045. Creating RedirectController and deserialization json from db

1. Creating RedirectController for all redirect logic
2. Creating method GetJsonFromDb
3. Creating deserialization

Nastsi committed:
issue#2045. Deleting RedirectController and making class RedirectMaker

Nastsi committed:
issue#2045. Deleting excess Compile Include Controllers

Nastsi committed:
issue#2045. Creating empty lines

jacob-l commented on RedirectApplication/RedirectApplication/RedirectMaker/Redirect.cs:
```diff
@@ -0,0 +1,25 @@
+
+        public DbJson DeserializationJsonFromDb(RedirectRule rule)
+        {
+            string json = db.GetJsonFromDb(rule.TargetUrl);
```
var

jacob-l commented on RedirectApplication/RedirectApplication/RedirectMaker/Redirect.cs:
```diff
@@ -0,0 +1,25 @@
+        {
+            string json = db.GetJsonFromDb(rule.TargetUrl);
+            var reader = new JsonTextReader(new StringReader(json));
+            DbJson content = JsonSerializer.CreateDefault().Deserialize<DbJson>(reader);
```
var

jacob-l replied:
return JsonSerializer...

Nastsi committed:
issue#2045 Var instead of string and return JsonSerializer

1. var json instead of string json
2. return JsonSerializer.CreateDefault().Deserialize<DbJson>(reader);
instead of return DbJson
